src/routes/teams/+page.svelte

```diff
@@ -8,7 +8,7 @@
 
 	<h2 class="sub-header-left">Add and change teams</h2>
 	<div class="NewTeams">
-		<Button Routing="Teams/new" LinkText="+ new team" />
+		<Button severity="primary-button" Routing="Teams/new" Text="+ new team" />
```

Comment:
Class `.primary-button` bestaat niet in het component. Het is by default sowieso een primary button, maar de class is `.primary`.